tools/rosgraph/src/rosgraph/roslogging.py

```diff
@@ -45,6 +45,13 @@
 
 class LoggingException: pass
 
+def renew_latest_logdir(logfile_dir):
+    log_dir = os.path.dirname(logfile_dir)
+    latest_dir = os.path.join(log_dir, 'latest')
+    if os.path.exists(latest_dir):
```

Comment:
I don't think this check will work if the symlink exists by points to a not existing folder.